_posts/2024-07-01-zio-fullstack-webapp.md

```diff
@@ -0,0 +1,844 @@
+---
+title: "ZIO Fullstack Webapp"
+date: 2024-07-05
+header:
+  image: "https://res.cloudinary.com/dkoypjlgr/image/upload/f_auto,q_auto:good,c_auto,w_1200,h_300,g_auto,fl_progressive/v1715952116/blog_cover_large_phe6ch.jpg"
+tags: [scala, zio, laminar, tapir]
+excerpt: "Full stack Scala webapp with ZIO, Laminar, and Tapir"
+toc: true
+toc_label: "In this article"
+---
+
+# Full stack ZIO Scala WebApp
+
+Leverage Scala / ScalaJS with ZIO and Laminar to build reactive webapp.
+
+Build a full stack webapp application using the same language has been a never ending quest. 
```

Comment:
Describe the app we're building in this article so that the reader knows what to expect. 1-2 sentences should be enough.